### DISCUSSION TITLE
how to run gguf

### DISCUSSION THREAD

**Discussion by kalle07** [mcmonkeyprojects/SwarmUI#384] [Q&A] [answered]
first time it install something but than have no idea

with ae-f16.ggu
16:32:39.605 [Info] ComfyUI backend 0 shutting down...
16:32:39.608 [Info] Shutting down self-start ComfyUI (port=7821) process #5352...
16:32:39.715 [Info] Self-Start ComfyUI-0 exited properly from disabling.
16:32:53.998 [Info] Self-Start ComfyUI (Requirements Install - ComfyUI-GGUF) exited properly.
16:32:54.001 [Init] Initializing backend #0 - ComfyUI Self-Starting...
16:32:56.035 [Init] Self-Start ComfyUI-0 on port 7822 is loading...
16:33:04.202 [Init] Self-Start ComfyUI-0 on port 7822 started.
== SwarmUI logs 2024-11-01 16:48 ==
16:48:21.646 [Info] User local requested 1 image with model 'flux1-dev-Q5_1.gguf'...
16:48:21.647 [Warning] [BackendHandler] No backends match the request! Cannot generate anything. Backends refused for the following reason(s):
- Request requires model 'ae-f16.gguf' but the backend does not have that model
16:48:21.648 [Error] [BackendHandler] Backend request #7 failed: No backends match the settings of the request given! Backends refused for the following reason(s):
- Request requires model 'ae-f16.gguf' but the backend does not have that model

with automatic vae
16:57:06.492 [Info] User local requested 1 image with model 'flux1-dev-Q5_1.gguf'...
16:57:06.502 [Error] [BackendHandler] backend #0 failed to load model with error: Model loader for flux1-dev-Q5_1.gguf didn't work - are you sure it has an architecture ID set properly?
16:57:06.503 [Warning] [BackendHandler] backend #0 failed to load model flux1-dev-Q5_1.gguf
16:57:07.501 [Warning] [BackendHandler] All backends failed to load the model 'e:\WebUI_Forge\webui\models\diffusion_models/flux1-dev-Q5_1.gguf'! Cannot generate anything.
16:57:07.501 [Error] [BackendHandler] Backend request #12 failed: All available backends failed to load the model 'e:\WebUI_Forge\webui\models\diffusion_models/flux1-dev-Q5_1.gguf'.
16:57:07.502 [Error] [BackendHandler] Backend request #12 failed: All available backends failed to load the model 'e:\WebUI_Forge\webui\models\diffusion_models/flux1-dev-Q5_1.gguf'.

all paremeter "default"
17:06:02.620 [Warning] [BackendHandler] backend #0 failed to load model flux1-dev-Q5_1.gguf
17:06:03.630 [Warning] [BackendHandler] All backends failed to load the model 'e:\WebUI_Forge\webui\models\diffusion_models/flux1-dev-Q5_1.gguf'! Cannot generate anything.
17:06:03.631 [Error] [BackendHandler] Backend request #7 failed: All available backends failed to load the model 'e:\WebUI_Forge\webui\models\diffusion_models/flux1-dev-Q5_1.gguf'.
17:06:03.631 [Error] [BackendHandler] Backend request #7 failed: All available backends failed to load the model 'e:\WebUI_Forge\webui\models\diffusion_models/flux1-dev-Q5_1.gguf'.

in FORGE it runs ....

the error messages icant get a hint where is the error ?!?

**Comment by mcmonkey4eva** [comment, marked as answer]
see gguf docs at https://github.com/mcmonkeyprojects/SwarmUI/blob/master/docs/Model%20Support.md#gguf-quantized-models

**Reply by kalle07** [reply]
i see
may on *.gguf files its possible to have an  error message like ... "look into Model Type Support In SwarmUI"  ;)
